### PR TITLE
Adapted and Added Functional and Non-Functional Requirements

### PR DESCRIPTION
**Pull request by m1cm1c**
Adapted functional and non-functional requirements to what's been discussed in the meeting of 2015-11-27 and added a new functional requirement.

### CODE REVIEW

File: doc/Requirements Specification/Parts/Non-Functional Requirements.lyx [hunk 1]
```diff
@@ -367,7 +351,7 @@ gls{SoMoX}
 
 \end_inset
 
-, such that it is automatically executed after 
+ to be executed after
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Non-Functional Requirements.lyx]: I think “automatically” is still important. Can you change it to “to automatically be executed after […]”?

Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Non-Functional Requirements.lyx]: done

**Comment by jGleitz** [comment]
lgtm

**Comment by viridispirus** [comment]
lgtm

**Pull request merged**